# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/emile/no-docs.patch
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- EMILE-EMILE-0.14/Makefile.vanilla	2024-03-27 13:10:43.357791000 +0000
+++ EMILE-EMILE-0.14/Makefile	2024-03-27 13:14:11.877791000 +0000
@@ -13,14 +13,14 @@
 
 .PHONY: first libemile libblock libiso9660 libiso9660-m68k libgzip-m68k tools \
        libcontainer clean all_bin all install tools-install first-install \
-       docs-install uninstall tools-uninstall first-uninstall docs-uninstall \
+       uninstall tools-uninstall first-uninstall \
        clean libemile-clean libmacos-clean libunix-clean tools-clean \
-       first-clean second-clean docs-clean libiso9660-clean libgzip-clean \
-       libfloppy-clean libscsi-clean libstream-clean libblock-clean dist docs \
+       first-clean second-clean libiso9660-clean libgzip-clean \
+       libfloppy-clean libscsi-clean libstream-clean libblock-clean dist \
        libconfig libconfig-m68k libmap \
        libmap-m68k libext2 libext2-m68k
 
-all: tools.mk docs libemile libblock libiso9660 libiso9660-m68k libgzip-m68k \
+all: tools.mk libemile libblock libiso9660 libiso9660-m68k libgzip-m68k \
      tools first libstream libcontainer libmap libext2 libext2-m68k \
      second/$(KARCH)-linux-floppy/second \
      second/$(KARCH)-linux-scsi/second second/m68k-netbsd-floppy/second \
@@ -267,21 +267,12 @@
 tools-uninstall::
 	$(MAKE) -C tools uninstall
 
-docs-install:: docs
-	$(MAKE) -C docs install
-
-docs-uninstall::
-	$(MAKE) -C docs uninstall
-
-docs::
-	$(MAKE) -C docs all
-
 dump: last.bin
 	dd if=last.bin of=$(FLOPPY) bs=512
 
-install: tools-install first-install second-install docs-install
+install: tools-install first-install second-install
 
-uninstall: tools-uninstall first-uninstall second-uninstall docs-uninstall
+uninstall: tools-uninstall first-uninstall second-uninstall
 
 libemile-clean:
 	$(MAKE) -C libemile clean
@@ -336,9 +327,6 @@
 second-clean:
 	$(MAKE) -C second clean
 
-docs-clean:
-	$(MAKE) -C docs clean
-
 libfloppy-clean:
 	$(MAKE) -C libfloppy clean
 
@@ -346,7 +334,7 @@
 	rm -f tools.mk
 
 clean:: libemile-clean libmacos-clean libunix-clean tools-clean first-clean \
-	second-clean docs-clean libiso9660-clean libext2-clean libgzip-clean \
+	second-clean libiso9660-clean libext2-clean libgzip-clean \
 	libfloppy-clean \
 	libscsi-clean libstream-clean libblock-clean libcontainer-clean \
 	libui-clean libconfig-clean libmap-clean
@@ -368,7 +356,6 @@
 	@$(MAKE) -C libemile dist DISTDIR=$(shell pwd)/$(PACKAGE)-$(VERSION)
 	@$(MAKE) -C second dist DISTDIR=$(shell pwd)/$(PACKAGE)-$(VERSION)
 	@$(MAKE) -C first dist DISTDIR=$(shell pwd)/$(PACKAGE)-$(VERSION)
-	@$(MAKE) -C docs dist DISTDIR=$(shell pwd)/$(PACKAGE)-$(VERSION)
 	@$(MAKE) -C libmacos dist DISTDIR=$(shell pwd)/$(PACKAGE)-$(VERSION)
 	@$(MAKE) -C libunix dist DISTDIR=$(shell pwd)/$(PACKAGE)-$(VERSION)
 	@$(MAKE) -C libui dist DISTDIR=$(shell pwd)/$(PACKAGE)-$(VERSION)
